### DISCUSSION TITLE
Problems with installation

### DISCUSSION THREAD

**Discussion by Rezipient** [typemill/typemill#480] [Q&A]
Hello, unfortunately I'm having trouble installing typemill. I unpacked the files and put the "cache", "content" etc. folders in a subfolder of my server.

Then I tried to visit the following URL:
https://mydomain.de/folder/setup

However, this results in a 404 page. And there is no file called setup.php or similar in the folders.

What did i do wrong?

I would be happy to receive some tips. Thank you!

Best regards

**Comment by trendschau** [comment]
Setup url is https://mydomain.de/folder/tm/setup (fixed it in the documentation) but it should redirect to the setup page anyway.

Are you able to visit any other content page, for example /getting-started or the homepage? 

For me it sounds like there is already a settings.yaml in the settings folder of your installation. If you want to setup the installation again, you have to go into the settings-folder and delete: 

* settings.yaml
* user (folder)

Then visit the page again and you should be redirected to the setup.
